Makefile

```diff
@@ -302,10 +302,10 @@ cmake-ios-framework:
 	cmake ../.. ${IOS_FRAMEWORK_CMAKE_PARAMS}
 
 ios-framework: cmake-ios-framework
-	xcodebuild -target ${IOS_FRAMEWORK_TARGET} -project ${IOS_FRAMEWORK_BUILD_DIR}/${IOS_FRAMEWORK_XCODE_PROJ}
+	xcodebuild -target ${IOS_FRAMEWORK_TARGET} -project ${IOS_FRAMEWORK_BUILD_DIR}/${IOS_FRAMEWORK_XCODE_PROJ} | xcpretty
 
 ios-sim: ${IOS_SIM_BUILD_DIR}/${IOS_XCODE_PROJ}
-	xcodebuild -target ${IOS_TARGET} -project ${IOS_SIM_BUILD_DIR}/${IOS_XCODE_PROJ}
+	xcodebuild -target ${IOS_TARGET} -project ${IOS_SIM_BUILD_DIR}/${IOS_XCODE_PROJ} | xcpretty
```

Comment:
Isn't `xcpretty` a default OS X installed command tool when xcode command line tools are installed? Otherwise yes. I added this for the convenience of reading travis builds, outputs there are pretty verbose :/